catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/util/impl/QueryResultIterable.java

```diff
@@ -53,7 +56,7 @@
      * @param filter  A dynamic supplier of a filter that takes the current index such that
```

Comment:
Which part of comments need changing? It appears that there weren't any param comments for the removed Function parameter.